src/test/java/HomepageTest.java

```diff
@@ -1,45 +1,146 @@
-import org.assertj.core.api.Assertions;
-import org.openqa.selenium.By;
+import org.assertj.core.api.SoftAssertions;
 import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
 import pages.Homepage;
 import utils.WebDriverUtils;
 
-import java.time.Duration;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
 public class HomepageTest {
 
     private WebDriver driver;
 
+    private static final String WHITE_COLOR = "#ffffff";
+    private static final String BLUE_COLOR = "#1979c3";
+
     @BeforeClass
     public void setUp() {
         driver = WebDriverUtils.createWebDriver();
     }
 
+    @BeforeMethod
```

Comment:
Same here why opening page takes place before each test?
If there is no page change during running tests in the class, then opening page should be done in the method annotated with @BeforeMethod